tsup.config.ts

```diff
@@ -5,15 +5,16 @@ import { defineConfig } from 'tsup';
 
 export default defineConfig((options: Options) => ({
   ...options,
-  entry: ['src/index.ts'],
-  treeshake: true,
+
+  cjsInterop: true,
+  clean: true,
+  dts: true,
+  entry: ['src/index.ts', 'src/lib/configure-security.ts'],
   format: ['esm', 'cjs'],
-  silent: !options.watch,
   minify: false,
-  dts: true,
+  shims: true,
+  silent: !options.watch,
   sourcemap: true,
-  clean: true,
   splitting: true,
```

Comment:
per [their docs](https://tsup.egoist.dev/#code-splitting), they already code split for ESM by default and this flag is enables an experimental feature where it attempts to split for CJS. i'm pretty hesitant to try that to be honest.

```suggestion
```